virtual-desktop-enhancer.ahk

```diff
@@ -232,6 +239,21 @@ OnShiftRightPress() {
     SwitchToDesktop(_GetNextDesktopNumber())
 }
 
+OnShiftLastActivePress() {
+    ; Shift to the last active desktop.
+
+    global lastActiveDesktopNumber
```

Comment:
You shouldn't have to identify that the variable is global if you already did it before when you defined it at the top of the file, can you look into this to make it work like it does for all of the other globals?